### DISCUSSION TITLE
X1Plus 1.0

### DISCUSSION THREAD

**Discussion by jwise** [X1Plus/X1Plus#75] [Release notes]
X1Plus 1.0 is our initial release of the X1Plus Custom Firmware for Bambu Lab X1 and X1 Carbon printers!  You can [grab it from the releases section](https://github.com/X1Plus/X1Plus/releases/tag/x1plus%2F1.0).

**Hardware and software supported**
This release supports X1 and X1 Carbon printers manufactured at least through March 2024.  X1E is not supported by this release.  This release requires version 01.06.06.58 or older of the Official Rootable Firmware to install.

**Features**
X1Plus 1.0 is our initial release, and comes with the following changes over the base 01.07.02.00 X1 firmware:

* WiFi performance improvements.
* Screen guard.
* Bed mesh visualization.
* Vibration calibration visualization.
* On-screen console.
* Persistent SSH keys (client and host).
* Printer statistics data visualization.
* Bed mesh and vibration calibration data are now exported over MQTT again.
* Firmware upgrade status for individual microcontroller boards.

For more information, see the X1Plus [installation guide](https://github.com/X1Plus/X1Plus/wiki/Installation-Guide) or [user's manual](https://github.com/X1Plus/X1Plus/wiki/User-Manual).

**Comment by h3r3andgon3** [comment]
The installer does not show any printers.  I've rebooted the PC and still no joy.  Any thoughts on why my printer isn't showing up?
